### PR TITLE
remove SSH development keys

### PR DESCRIPTION
**Pull request by Stadicus**
fixes https://github.com/digitalbitbox/bitbox-base/issues/292

The SSH login can now be reliably set using the BitBoxApp, so untrusted SSH keys no longer need to be part of the official repo.

Users can still add their own keys in `armbian/base/config/ssh/authorized_keys`, which is excluded from Git versioning. To include these keys, also set `BASE_ADD_SSH_KEYS='true'` in `build.conf`.

This pull request
* removes the Shift ssh dev keys in `authorized_keys`
* adds `authorized_keys.readme` with additional information
* adds `authorized_keys` to `.gitignore`

### CODE REVIEW

File: armbian/base/config/ssh/authorized_keys.readme [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# For secure SSH login, add your SSH key to the file `authorized_keys` in this directory.
+# It is excluded from Git versioning control.
+#
+# See this introduction for additional information: <https://www.atlassian.com/git/tutorials/git-ssh>
```
Comment (x1ddos) [inline armbian/base/config/ssh/authorized_keys.readme]: Why not simply add the contents of this file above `BASE_ADD_SSH_KEYS` option in build.conf?
This file is much harder to discover compared to reading the doc comment in the config file.

Comment (Stadicus) [reply armbian/base/config/ssh/authorized_keys.readme]: I thougt about that but think this has its limits when working with multiple keys. Would work technically, I guess, but I'm not really comfortable with multi-line variables.

Comment (x1ddos) [reply armbian/base/config/ssh/authorized_keys.readme]: Multi-line variables? What I meant is this, in the `build.conf` file:

```
# Add SSH keys in ./config/ssh/authorized_keys file for remote login.
# The file is excluded from Git versioning control.
# See this introduction for additional information: <https://www.atlassian.com/git/tutorials/git-ssh>
#BASE_ADD_SSH_KEYS="false"
```

Comment (Stadicus) [reply armbian/base/config/ssh/authorized_keys.readme]: Ah, I read that wrongly and thought about adding the keys themselves to the build.conf. Good point, will add the info here.

Comment (Stadicus) [reply armbian/base/config/ssh/authorized_keys.readme]: User can now add own keys in `armbian/base/config/authorized_keys`, instructions are moved to `build.conf`

**Pull request merged**